composed_api.rst

```diff
@@ -287,20 +276,20 @@ Endpoint
 
 ::
 
-    GET <instance_url>/<instance_root>_ba/api/v1/aoi/add
+    GET <instance_url>/<instance_root>_ba/api/v2/aoi/add
 
 Request Parameters
 ^^^^^^^^^^^^^^^^^^
 
 +-------------------+-------------------------------------------------------+
 | Query parameter   | Value                                                 |
 +===================+=======================================================+
+| source            | *Required*. Your GRiD generated API key.              |
```

Comment:
Well, maybe I was just wrong. I thought the key could only be used by the person who created it.